documentation/docs/welcome.md

```diff
@@ -24,6 +24,12 @@ protect any digital secret.
 Stronghold is a secure database for working with cryptography, which ensures that secrets (like private keys) are never
 revealed, but can be used according to best practices.
 
+Stronghold can be seen as some kind of isolated container for secrets that may be either a private key, or some other bytes of data, that should never be exposed. In order to work with Stronghold one uses  so called procedures to either generate new keys, store data into the vault, derive keys or work with the data in place. The difference to a "traditional" password store is, that secret data is never accessed directly but can be worked with the aforementioned procedures. Whenever there is a need to sign some data with a private key, one would be required to call a respective function to work with it. 
```

Comment:
```suggestion
You can think of Stronghold as an isolated container for secrets. The secret can be a private key or other bytes of data that should never be exposed. You can interact with Stronghold using procedures to [generate new keys](how_tos/cli/generate_key_pair), [store data in the vault](how_tos/cli/store_read_write), [derive keys](how_tos/cli/derive_slip_10), or work with the data. The difference to a "traditional" password store is that you can never access the secret data directly. You will need to use procedures. For example, if you need to sign data with your private key, you must call a procedure.
```

Reply:
I was using "you" before and changed it to be more neutral :sweat_smile: But I do see the point.